sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -39,6 +36,8 @@ import (
 	"github.com/stackrox/rox/sensor/common/message"
 	"github.com/stackrox/rox/sensor/common/metrics"
 	flowMetrics "github.com/stackrox/rox/sensor/common/networkflow/metrics"
+	"golang.org/x/text/cases"
```

Comment:
This reordering looks wrong. 🤔

Reply:
seems right to me? Before, there were three separate import blocks, but we usually just have two (stlib and "else"). For some reason, there were three. Running `golangci-lint` ended up fixing this, though it does seem unrelated to the rule I added 🤷

Reply:
Ah, I misread the change, and also thought we wanted 3 blocks. meh 🤷🏻